src/routes/+page.svelte

```diff
@@ -88,36 +82,27 @@
 <ChocoSection>
   <div class="flex flex-col items-start md:flex-row md:items-center">
     <div class="w-full md:w-3/5">
-      <SectionH2>¿Quiénes<wbr /> Somos?</SectionH2>
+      <SectionH2>Choco<wbr />Misión</SectionH2>
       <p class="mt-3 text-xl">
-        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Error repudiandae animi
-        necessitatibus maiores, quis temporibus recusandae fuga exercitationem iste nesciunt sint
-        culpa, tempora doloremque! Qui expedita sint voluptate non et.
+        Compartir, difundir e implementar, conocimiento y herramientas, para el desarrollo académico
+        y tecnológico de la sociedad Ecuatoriana.
       </p>
     </div>
-    <div class="my-8 mx-auto aspect-[4/5] w-40 bg-zinc-500" />
-  </div>
-</ChocoSection>
-<ChocoSection>
-  <SectionH2>Choco<wbr />misión</SectionH2>
-  <div class="flex flex-col gap-8 sm:grid sm:grid-cols-3">
-    {#each chocomisions as content, idx}
-      <Chocomision {content} {idx} />
```

Comment:
We don't use this component anymore. Let's delete the import

Reply:
done